VERSION

```diff
@@ -1 +1 @@
-0.15.0
+0.16.0
```

Comment:
this is a breaking change... people using this lib from a project using python < 3.8 will be broken, let's increment the major version then

Reply:
reverted the commit, will create new MR with major version bump

Reply:
thx for making this a major. We are working in moving qontract-reconcile to a newer Python version, still not there 😓